app/view/component/MapController.js

```diff
@@ -14,6 +14,7 @@ Ext.define('momo.view.component.MapController', {
             var olMap = me.createOlMap();
             view.setMap(olMap);
         }
+        olMap.on('moveend',me.setRouting, me);
```

Comment:
No, not yet. Is it really necessary to unregister it till we still have a pretty thin application without login/logout, sessions etc.?

I will keep it in mind for the future and possibly add it later.